Apps/WebClient/test/functional/e2e/cypress.json

```diff
@@ -16,8 +16,12 @@
   "env": {
     "bcsc.username": "hthgtwy11",
     "bcsc.password": "",
+    "bcsc.hlthgw401.username": "hlthgw401",
+    "bcsc.hlthgw401.password": "",
     "keycloak.username": "healthgateway",
     "keycloak.password": "",
+    "keycloak.healthgateway12.username": "healthgateway12",
```

Comment:
switched to keycloak user for testing the "Client Registration error" scenario per @sslaws 's suggestions.